releasenotes/2.1.0.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.0
+date: 2018-02-21 12:00:00
+release_notes: true
+permalink: /releasenotes/2.1.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.0 represents an important milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](https://s.apache.org/jclouds210). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.1.0 features include:
+
+* Modernize Openstack support
+* Graduate Microsoft Azure Resource Manager to core
+* Better support for AWS EC2 in terms of network capabilities and regions
+* Graduate Google Cloud Storage (GCS) and Backblaze B2 to core
+* Better integration with OSGi and Apache Karaf.
+* Numerous bug fixes and performance improvements.
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314430&version=12334714).
+
+## <a id="computeimprovements"></a>Compute Improvements
+
+* Add Support OpenStack Keystone v3 API
+* Promote Openstack Neutron to core
+* Better support of Openstack Neutron from Openstack Nova
```

Comment:
Link to the context linking blog post (I have to finish it).

Reply:
good idea. FYI I've put a placeholder `/blog/2018/01/16/keystone-v3/` for it